views/pages/userPage/creatorPage.ejs

```diff
@@ -52,134 +59,83 @@
+                <h3 class="item-title"><%= item.name %></h3>
+                <p class="item-price"><%= item.price === 0 ? 'FREE' : '$' + item.price %></p>
+                <div class="content-actions">
+                  <a href="/pages/userPage/EditCreation?itemNumber=<%= item.itemNumber %>" class="btn btn-info" role="button">Edit</a>
+                  <button class="btn btn-danger delete-btn" data-item-number="<%= item.itemNumber %>">Delete</button>
+                </div>
               </div>
-            </div>
-          </a>
-          <div class="content-download">
-            <a href="/pages/userPage/EditCreation" class="btn btn-info" role="button">Edit</a>
-          </div>
-        </div>
-      </div>
-    </section>
-
-    <div class="divider"></div>
 
-    <section class="content-section">
-      <h2 class="text-center">Unpublished Content</h2>
-      <div class="row content-viewer">
-        <div class="col-sm-4">
-          <a href="/pages/itemPages/companionAnimalsExpanded" class="thumbnail-link">
-            <div class="thumbnail">
-              <img src="/images/Lamia_Tera_Milteir.webp" alt="Monstrous Myths">
-              <div class="caption">
-                <h3>Monstrous Myths</h3>
-                <p>$1.99</p>
-              </div>
             </div>
-          </a>
-          <div class="content-download">
-            <a href="/pages/userPage/EditCreation" class="btn btn-info" role="button">Edit</a>
           </div>
-        </div>
-        <div class="col-sm-4">
-          <a href="/pages/itemPages/companionAnimalsExpanded" class="thumbnail-link">
-            <div class="thumbnail">
-              <img src="/images/elemental.png" alt="Elementals ETC">
-              <div class="caption">
-                <h3>Elementals etc</h3>
-                <p>3.99</p>
-              </div>
-            </div>
-          </a>
-          <div class="content-download">
-            <a href="/pages/userPage/EditCreation" class="btn btn-info" role="button">Edit</a>
-          </div>
-        </div>
-        <div class="col-sm-4">
-          <a href="/pages/itemPages/companionAnimalsExpanded" class="thumbnail-link">
+          <% if ((index + 1) % 4 === 0) { %>
+            </div><div class="row content-viewer">
+          <% } %>
+        <% } %>
+      <% }); %>
+    </div>
+  <% } else { %>
+    <h4 class="text-center">No published items yet.</h4>
+  <% } %>
+</section>
+
+<!-- Unpublished Content Section -->
+<section class="content-section">
+  <h2 class="text-center">Unpublished Content</h2>
+  <% if (authorName && unpublishedItems && unpublishedItems.length > 0) { %>
+    <div class="row content-viewer">
+      <% unpublishedItems.forEach((item, index) => { %>
+        <% if (item.ownedByAuthor) { %>
+          <div class="col-sm-3">
             <div class="thumbnail">
-              <img src="/images\beach.jpg" alt="outdoor encounters">
+              <a href="/pages/item/<%= item.itemNumber %>" class="thumbnail-link">
```

Comment:
this line should be
<a href="/item/<%= item.itemNumber %>" class="thumbnail-link">

Reply:
<a href="/item/<%= item.itemNumber %>" class="thumbnail-link">